content/_guides/sasl/emacs-erc.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Configuring SASL for Emacs ERC
+category: sasl
+credits: androclus
+---
+
+[ERC](https://www.emacswiki.org/emacs/ERC) is an IRC client which comes as a package for Emacs.
+
+## A. Set up ERC IRC client
+
+If you have ERC already installed and running on your emacs system, then skip to part B.
+Otherwise, set up your ERC with basic functionality, as follows:
+
+1. Download and Install the erc package from melpa, etc. (*M-x package-list-packages*)
```

Comment:
Should this perhaps be escaped as code/commands rather than emphasised?
```suggestion
1. Download and Install the erc package from melpa, etc. (`M-x package-list-packages`)
```

Reply:
okay, changed as you recommend